web/src/js/ducks/ui.js

```diff
@@ -0,0 +1,37 @@
+import { SELECT_FLOW } from './flows'
+const SET_ACTIVE_MENU = 'SET_ACTIVE_MENU'
+
+
+const defaultState = {
+    active_menu: 'Start'
+}
+export default function reducer(state = defaultState, action) {
+    switch (action.type) {
+        case SET_ACTIVE_MENU:
+            return {
+                ...state,
+                active_menu: action.active_menu
+            }
+        case SELECT_FLOW:
+            let isNewSelection = (action.flowId && !action.currentSelection)
+            let isSelectAction = action.flowId
```

Comment:
What's the point of `isSelectAction` when `isNewSelection` already assures that `action.flowId` is true? The `false` case can never get triggered currently.